test/api/publish.test.js

```diff
@@ -44,35 +49,20 @@ describe('publishing endpoints', () => {
     return agent.post('/compressed-store').accept(acceptHeader).type('text');
   }
 
-  it('uploads data all the way to google storage when using a content length', async () => {
```

Comment:
This didn't disappear, this is just below. But I extracted quite a big part of this test in a function, that's why github is showing it like this.